policyengine_us/parameters/gov/states/nm/tax/income/credits/ctc/amount.yaml

```
@@ -6,11 +6,15 @@ metadata:
   label: New Mexico child income tax credit amount
   reference:
     - title: New Mexico Statutes, 7-2-18.34. Child income tax credit.
-      href: https://nmonesource.com/nmos/nmsa/en/item/4340/index.do#!fragment/zoupio-_Toc140503818/BQCwhgziBcwMYgK4DsDWszIQewE4BUBTADwBdoAvbRABwEtsBaAfX2zgEYAWABgFYeAZgAcHYQEoANMmylCEAIqJCuAJ7QA5BskRCYXAiUr1WnXoMgAynlIAhdQCUAogBknANQCCAOQDCTyVIwACNoUnZxcSA
+      href: https://nmonesource.com/nmos/nmsa/en/item/4340/index.do#!fragment/zoupio-_Toc193384849/BQCwhgziBcwMYgK4DsDWszIQewE4BUBTADwBdoAvbRABwEtsBaAfX2zgEYBOAZh4A4ALEK4BKADTJspQhACKiQrgCe0AORrxEQmFwIFS1Rq069IAMp5SAIVUAlAKIAZBwDUAggDkAwg-GkwACNoUnZRUSA
     - title: New Mexico Session Laws, Laws 2023 - First Session, Fifty-Sixth Legislature, Chapter 211 Section 9 Laws 2023
-      href: https://nmonesource.com/nmos/nmsl/en/item/18775/index.do#!fragment/undefined/BQCwhgziBcwMYgK4DsDWsBGB7LqC2YATqgJIAm0cATAIw0QCcAlADTJYAuAphAIqJdCAT2gByUSwhciCfoJHjJ0wggDKWQhwBCIgEoBRADL6AagEEAcgGF9LDmAzQOWOEyZA
+      href: https://nmonesource.com/nmos/nmsl/en/item/18775/index.do#!fragment/undefined/BQCwhgziBcwMYgK4DsDWsBGB7LqC2YATqgJIAm0cATAIw0QCcAlADTJYAuAphAIqJdCAT2gByUSwhciCfoJHjJ0wggDKWQhwBCIgEoBRADL6AagEEAcgGF9LDmAzQOWOEyZA   
+    - title: New Mexico Statutes, 7-2-18.34(D) - Inflation adjustment
+      href: https://nmonesource.com/nmos/nmsa/en/item/4340/index.do#!fragment/zoupio-_Toc193384849/BQCwhgziBcwMYgK4DsDWszIQewE4BUBTADwBdoAvbRABwEtsBaAfX2zgEYBOAZh4A4ALEK4BKADTJspQhACKiQrgCe0AORrxEQmFwIFS1Rq069IAMp5SAIVUAlAKIAZBwDUAggDkAwg-GkwACNoUnZRUSA
```

This link took me straight there.
```suggestion
      href: https://nmonesource.com/nmos/nmsa/en/item/4340/index.do#!fragment/zoupio-_Toc194323097/BQCwhgziBcwMYgK4DsDWszIQewE4BUBTADwBdoAvbRABwEtsBaAfX2zgEYBOAFgGYATHwAMXAOwBKADTJspQhACKiQrgCe0AOSapEQmFwJlqjdt37DIAMp5SAIQ0AlAKIAZZwDUAggDkAws5SpGAARtCk7BISQA
```